index.js

```diff
@@ -245,12 +245,16 @@ module.exports = {
       destDir: '@babel/runtime/helpers/esm'
     });
 
-    return this.transpileTree(babelHelpersTree, {
+    const transpiledHelpers = this.transpileTree(babelHelpersTree, {
       'ember-cli-babel': {
         // prevents the helpers from being double transpiled, and including themselves
         disablePresetEnv: true
       }
     });
+
+    return new Funnel(transpiledHelpers, {
+      destDir: this.moduleName(),
+    });
```

Comment:
Doesn't this mean that the modules aren't in the right place when they are imported? Or maybe the answer is: it doesn't matter because the define statements (created by `this.transpileTree` just above) will the right module names (unscoped to `ember-cli-babel`).

Would you mind double checking in a `ember new` style app with and without Embroider (with the flag turned on) to confirm that things work properly (and the modules end up in the build with the right module paths)?

Reply:
I dug into the output and confirmed that this is indeed how it works. Sorry for the noise @charlespierce.